install_pim/docker/installation_docker.rst

```diff
@@ -159,13 +152,13 @@ If you want to execute only one job:
 Xdebug
 ******
 
-*Xdebug* is deactivated by default. If you want to activate it, you can use the environment variable ``XDEBUG_ENABLED`` and set its value to 1.
+*Xdebug* is disabled by default. If you want to enable it, set the environment variable ``XDEBUG_ENABLED`` to `1`.
 
 Enable it on fpm service:
 
 .. code-block:: bash
 
-    $ XDEBUG_ENABLED=1 make up
+    $ XDEBUG_ENABLED=1 make dev
```

Comment:
```suggestion
    $ XDEBUG_ENABLED=1 make pim-dev
```

Reply:
Nope, it's not the same Makefile. It does more than just pim-dev or pim-prod ;)